src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -8,5 +8,7 @@ public-f net.minecraft.client.AbstractOption func_243220_a()Lnet/minecraft/util/
 public-f net.minecraft.client.AbstractOption func_243225_c(I)Lnet/minecraft/util/text/ITextComponent; # getMessageWithValue
 public-f net.minecraft.client.AbstractOption func_243222_a(Lnet/minecraft/util/text/ITextComponent;)Lnet/minecraft/util/text/ITextComponent; # getGenericValueComponent
 public-f net.minecraft.client.gui.widget.TextFieldWidget func_212951_d(Ljava/lang/String;)V # onTextChanged
+public-f net.minecraft.client.renderer.texture.SpriteUploader func_215282_a(Lnet/minecraft/util/ResourceLocation;)Lnet/minecraft/client/renderer/texture/TextureAtlasSprite; # getSprite (SpriteUploader)
+protected-f net.minecraft.client.renderer.entity.PaintingRenderer func_229122_a_(Lcom/mojang/blaze3d/matrix/MatrixStack;Lcom/mojang/blaze3d/vertex/IVertexBuilder;Lnet/minecraft/entity/item/PaintingEntity;IILnet/minecraft/client/renderer/texture/TextureAtlasSprite;Lnet/minecraft/client/renderer/texture/TextureAtlasSprite;)V
```

Comment:
where comment

Reply:
the function has no mapping name, its just called func_229122_a_